### PR TITLE
Describe depth-stencil formats

### LINKED ISSUE #24: Depth/stencil textures support
We need to verify whether KTX2 needs to support depth and/or stencil textures.

If it does:
- Which texture types could be used with D/S data?
- How does swizzling interact with D/S?

### PR DESCRIPTION
**Pull request by lexaknyazev**
Fixes #24.

### CODE REVIEW

File: ktxspec.adoc [hunk 1]
```diff
@@ -400,6 +400,32 @@ specification as the formats are not in wide use and applications
...
+value in the eight LSBs and zeros in the eight MSBs.
+
+`VK_FORMAT_D24_UNORM_S8_UINT` is defined as one 32-bit word per texel
+with the S8 value in the eight LSBs of the word and the D24 value in the MSBs.
+
+`VK_FORMAT_D32_SFLOAT_S8_UINT` is defined as two 32-bit words per texel.
+The first word contains the floating-point D32 value. The second word contains
+the S8 value in the eight LSBs and zeros in the MSBs.
+
+`VK_FORMAT_S8_UINT`, `VK_FORMAT_D16_UNORM`, `VK_FORMAT_X8_D24_UNORM_PACK32`,
```
Comment (MarkCallow) [inline ktxspec.adoc]: Shouldn't we say the X8 bits should be 0?

**Comment by lexaknyazev** [comment]
From https://github.com/KhronosGroup/KTX-Specification/pull/61#issuecomment-465592355:
> I think for all of them `typeSize` should be 4.

That would affect packing of `VK_FORMAT_D16_UNORM_S8_UINT`. We should decide whether it's one 32-bit word or two 16-bit words. In the latter case, `typeSize` would be 2.

File: ktxspec.adoc (hunk 1 above)
Comment (lexaknyazev) [reply ktxspec.adoc]: Fixed.

**Comment by MarkCallow** [comment]
Thank you.

**Pull request merged**